www/templates/activity/post-activity.html

```diff
@@ -36,7 +36,9 @@
 			  <a class="item item-thumbnail-left item-icon-right"  ng-click="goToAttachPicture()" ng-if="allowImageUpload && pictureUploaded!=null">
 			    <img ng-src="{{pictureUploaded}}">
 			    <i class="icon ion-android-cloud-done balanced"></i>
-			  </a>			  
+			  </a>		
+			  <ion-checkbox ng-model="pushNotifs.onlyToHomeOwners">Notify home owners only?</ion-checkbox>
+			  <ion-checkbox ng-if="regionSettings.multiBlock" ng-model="pushNotifs.onlyToMyBlock">Notify your block only?</ion-checkbox>
```

Comment:
Option should be, "Want to send this message to specific block?". When they select this option, we need to show the list of blocks in that community and send the message to the selected block. This is because admins wanted to use this functionality to send messages to specific blocks.